### PR TITLE
Context file names refactoring

### PR DESCRIPTION
**Pull request by RomanSavran**
- Changes to Buildings according to Sebastian update
- Changing file names to snake case

### CODE REVIEW

**Comment by RomanSavran** [comment]
@soderluk could you please check and merge request? I changed context file names and several requests by Sebastian. DLI ontology is in progress

File: contexts/identity-building_commenrcia_wholesale.jsonld [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{
```
Comment (soderluk) [inline contexts/identity-building_commenrcia_wholesale.jsonld]: This filename? What is commenrcia? Please run a spell check on all filenames. I can't really review 900 filename changes in one PR.

File: contexts/identity-building_commenrcial_retail.jsonld [hunk 2]
```diff
@@ -0,0 +1,80 @@
+{
```
Comment (soderluk) [inline contexts/identity-building_commenrcial_retail.jsonld]: Same here. Filename typo

File: contexts/identity-building_commenrcial_retail_shopping_center.jsonld [hunk 3]
```diff
@@ -0,0 +1,80 @@
+{
```
Comment (soderluk) [inline contexts/identity-building_commenrcial_retail_shopping_center.jsonld]: Same here

**Review by soderluk — CHANGES_REQUESTED** [review]
At least some of the filenames had typos in them, please fix them, and at least run some kind of spellcheck on the files.

File: contexts/identity-building_commenrcia_wholesale.jsonld (hunk 1 above)
Comment (RomanSavran) [reply contexts/identity-building_commenrcia_wholesale.jsonld]: These examples are enough. I relied on new building structure input from Excels provided without spellcheck... Fixing

Comment (RomanSavran) [reply contexts/identity-building_commenrcia_wholesale.jsonld]: @soderluk Please re-check

**Comment by soderluk** [comment]
@RomanSavran Please abandon this PR as it's redundant and will be replaced with the new features.

**Pull request closed without merging**